src/servers/esl.js

```diff
@@ -13,15 +13,19 @@ class EslServer extends SocketServer {
   }
 
   holdCall(callId) {
+    this._getSocket(callId).event_json('CHANNEL_HANGUP');
+    this._getSocket(callId).filter('variable_conference_name', callId);
```

Comment:
https://github.com/utilitywarehouse/uw-lib-callcontrol/pull/2#discussion_r133983969

freeswitch prefixes it automatically!:)